terraform/platforms/aws/output.tf

```diff
@@ -20,3 +20,11 @@ output "etcd_address" {
 output "instance_security_group" {
   value = "${aws_security_group.instances.id}"
 }
+
+output "elb_dns_name" {
```

Comment:
nit: maybe name it "lb" to make it more generic in case we change the underlying

Reply:
updated name to "lb"